src/table/internal.tsx

```diff
@@ -243,7 +243,11 @@ const InternalTable = React.forwardRef(
             )}
             <table
               ref={tableRef}
-              className={clsx(styles.table, resizableColumns && styles['table-layout-fixed'])}
+              className={clsx(
+                styles.table,
+                resizableColumns && styles['table-layout-fixed'],
+                hasFooter && styles['has-footer']
```

Comment:
Does the `has-footer` class do anything on the `table` element? I couldn't find any selector in the styles that would match this.

Reply:
Good catch. It looks like it is unused.